### DISCUSSION TITLE
Vercel deployment error

### DISCUSSION THREAD

**Discussion by TheusMarqs** [vercel/community#6594] [Help] [answered]
### Summary

I am having some problems with my django app deployments on vercel

### Steps to Reproduce

A few days ago my deployment on Vercel started giving this error:

This serverless function has crashed.

Your connection is working properly.

Vercel is working properly.

I returned my code to one that was working, but this error was permanent, even then with the same code as a deploy that was working. So, I researched and found someone who had this problem and solved it by changing vercel.json, adding "version": 2 at the top, like this:

```
{
    "version 2,
    "builds": [
        {
            "src": "core/wsgi.py",
            "use": "@vercel/python",
            "config": { "maxLambdaSize": "15mb", "runtime": "python3.9" }
        }
    ],
    "routes": [
        {
            "src": "/(.*)",
            "dest": "core/wsgi.py"
        }
    ]
}
```

So I did that and my deploy worked.

But now it has stopped working again, even though the code is exactly the same as a deploy that was working. Can anyone help me solve this problem?

**Comment by QuiiBz** [comment, marked as answer]
👋 Could you try to re-deploy your project now? The issue should be fixed.

**Reply by TheusMarqs** [reply]
yes, it's working! thank you
